utils/config/service_configuration_test.go

```diff
@@ -333,3 +333,84 @@ func TestFlagBindingDefaults(t *testing.T) {
 	assert.Equal(t, DefaultConfiguration().TestConfig.Flag, configTest.TestConfig.Flag)
 	assert.Equal(t, DefaultConfiguration().TestConfig.Flag, configTest.TestConfig2.Flag)
 }
+
+// Test you can use a struct to load the default env vars
+func TestGenerateEnvFile_Defaults(t *testing.T) {
+	configTest := DefaultDummyConfiguration()
+	prefix := "test"
+
+	// Create test data
+	testValues := map[string]interface{}{
+		"TEST_DB":                 configTest.DB,
+		"TEST_DUMMY_HOST":         configTest.Host,
+		"TEST_FLAG":               configTest.Flag,
+		"TEST_HEALTHCHECK_PERIOD": configTest.HealthCheckPeriod,
```

Comment:
could you create a multiple layer configuration object
i.e. config1struct{
config2struct{
config3field1...
}
}